### PR TITLE
fixed auth in wish list controller

### PR DESCRIPTION
**Pull request by AinaArd**

### CODE REVIEW

File: src/main/java/ru/itis/controllers/WishListController.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public WishListController(ItemService itemService, WishListService wishListServi
     @GetMapping("/lists/{list-id}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Get a single wish list")
-    public WishListDto getWishList(@PathVariable Long listId) {
+    public WishListDto getWishList(@PathVariable("list-id") Long listId, @RequestHeader(name = "AUTH") String token) {
```
Comment (Dimanaux) [inline src/main/java/ru/itis/controllers/WishListController.java]: Do we need this unused parameter `@RequestHeader(name = "AUTH") String token`?

Comment (Dimanaux) [inline src/main/java/ru/itis/controllers/WishListController.java]: If we use `@GetMapping("/lists/{listId}")` and param `@PathVariable Long listId`, does it work?

File: src/main/java/ru/itis/controllers/WishListController.java [hunk 2]
```diff
@@ -41,15 +41,16 @@ public WishListDto getWishList(@PathVariable Long listId) {
     @PostMapping("/lists/{list-id}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Add new item to wish list")
-    public void addNewItem(@PathVariable Long listId, ItemForm itemForm) {
+    public void addNewItem(@PathVariable("list-id") Long listId, ItemForm itemForm, @RequestHeader(name = "AUTH") String token) {
```
Comment (Dimanaux) [inline src/main/java/ru/itis/controllers/WishListController.java]: See comments above.

File: src/main/java/ru/itis/controllers/WishListController.java (hunk 1 above)
Comment (AinaArd) [reply src/main/java/ru/itis/controllers/WishListController.java]: If we allow only authorized users to enter this page, then yes

Comment (AinaArd) [reply src/main/java/ru/itis/controllers/WishListController.java]: It doesn't work without this param

**Pull request merged**